analyzer/tests/functional/analyze/test_analyze.py

```diff
@@ -712,3 +712,50 @@ def test_multiple_invalid_checker_names(self):
 
         errcode = process.returncode
         self.assertEqual(errcode, 0)
+
+    def test_analyzer_and_checker_config(self):
+        build_json = os.path.join(self.test_workspace, "build_success.json")
+        source_file = os.path.join(self.test_dir, "tidy_report.cpp")
+        build_log = [{"directory": self.test_workspace,
+                      "command": "gcc -c " + source_file,
+                      "file": source_file}]
+
+        with open(build_json, 'w',
+                  encoding="utf-8", errors="ignore") as outfile:
+            json.dump(build_log, outfile)
+
+        analyze_cmd = [self._codechecker_cmd, "check", "-l", build_json,
+                       "--analyzers", "clang-tidy", "-o", self.report_dir,
```

Comment:
Could you please add a new test case or extend this one and enable clangsa analyzer and add some analyzer and checker config for the analyze command to make sure that it works properly with this analyzer too.